apps/src/netsim/NetSimMessage.js

```diff
@@ -59,7 +62,8 @@ var NetSimMessage = module.exports = function (shard, messageRow) {
    * All other message content, including the 'packets' students will send.
    * @type {*}
    */
-  this.payload = messageRow.payload;
+  var b64 = messageRow.base64Payload;
+  this.payload = (b64) ? base64ToBinary(b64.string, b64.len) : undefined;
```

Comment:
Style nits:
* Characters are cheap and our stuff gets minified anyway; maybe `s/b64/base64Payload`
* Parentheses around b64 in your ternary expression are unnecessary.